src/RateTargetKinkInterestRateStrategy.sol

```diff
@@ -10,9 +10,7 @@ import {IDefaultInterestRateStrategy} from 'aave-v3-core/contracts/interfaces/ID
 import {IReserveInterestRateStrategy} from 'aave-v3-core/contracts/interfaces/IReserveInterestRateStrategy.sol';
 import {IPoolAddressesProvider} from 'aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol';
```

Comment:
Align, change `'` to `"` across the whole repo.

Reply:
fixed